### ISSUE TITLE
Api call (Rest IP) -> "make this the primary ip for the device/vm" in IP address (select/unselect)

### ISSUE THREAD

**Issue by lavazza2017** [netbox-community/netbox#17147]
### NetBox version

v4.0.8

### Feature type

Change to existing functionality

### Proposed functionality

select or unselect (default) via api call "make this the primary ip for the device/vm" in IP address
I didn't find any possibility to send api call (post) to change this state.

### Use case

Most of my and most likely people need this selection

**Comment by arthanson** [comment]
@lavazza2017 can you please provide more detail about what you are wanting?

Unfortunately, the information you have provided does not constitute an actionable feature request. Per our [contributing guide](https://github.com/netbox-community/netbox/blob/develop/CONTRIBUTING.md), a feature request must include a thorough description of the proposed functionality, including any database changes, new views or API endpoints, and so on. It must also include a detailed use case justifying its implementation. If you would like to elaborate on your proposal, please modify your post above. If sufficient detail is not added, this issue will be closed.

**Comment by lavazza2017** [comment]
Hmm, sorry, I'm not so technically savvy that I could give you everything on a plate :(

The description is pretty clear. But I'll try again in other words.

I make all entries to Netbox automatically through the API call Post
http://netbox/api/ipam. All options are there "Tenant", "Interface" etc. and everything I can add, unfortunately I didn't find the option like in the topic "make this the primary ip for the device/vm" I also added a screenshot of what the option looks like from the gui.
![image](https://github.com/user-attachments/assets/7a69f584-382b-4919-8589-fd471ae4291e)

I'm sorry I don't know how to add an option to the database or to the rest ip, I'm just an ordinary user who uses the available options. And here one is missing, so I initiated adding it
